localenv/happy-life-bank/docker-compose.yml

```diff
@@ -69,6 +71,13 @@ services:
       KEY_ID: 53f2d913-e98a-40b9-b270-372d0547f23d
     depends_on:
       - cloud-nine-backend
+    healthcheck:
+      test: ["CMD", "wget", "--header=apollo-require-preflight: true", "http://localhost:3001/graphql?query=%7B__typename%7D", "-O", "/dev/null"]
```

Comment:
To confirm, this is just making an empty GQL request > what does this header do?

Reply:
> To confirm, this is just making an empty GQL request > what does this header do?

Yes, empty gql request. the header is to ensure CSRF protection doesnt block it. Took this request from apollo server docs here which mentions the header: https://www.apollographql.com/docs/apollo-server/monitoring/health-checks/